README.md

```diff
@@ -746,6 +746,12 @@ Examples are better than a long text, so here you go:
 #### General
 
 - Define your config template in the main lovelace configuration and then use it in your button-card. This will avoid a lot of repetitions! It's basically YAML anchors, but without using YAML anchors and is very useful if you split your config in multiple files 😄
+  Make sure which type of lovelace you are using before changing main lovelace configuration:
```

Comment:
Some corrections:
`Make sure which type of lovelace dashboard you are using before changing the main lovelace configuration:`